package.json

```diff
@@ -117,10 +117,10 @@
     ],
     "coverageThreshold": {
       "global": {
-        "branches": 90,
-        "functions": 90,
-        "lines": 90,
-        "statements": 90
+        "branches": 80,
```

Comment:
according to https://codecov.io/gh/leapdao/leap-node/compare/127fe7c2293f82a19af71700f5aee2ab12068bd7...cffdf6ea17406276d68b2aa8404cfee8f1afaaa5 we are still hitting 90%, no?